CMakeLists.txt

```diff
@@ -559,36 +567,15 @@ if (VITA_BUILD)
 
     vita_create_self(${SHORT_NAME}.self ${SHORT_NAME} UNSAFE UNCOMPRESSED)
     vita_create_vpk(${SHORT_NAME}.vpk ${VITA_TITLEID} ${SHORT_NAME}.self
-        VERSION ${VITA_VERSION}
+        VERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}${VERSION_REVISION}"
```

Comment:
Why not use `VITA_VERSION`?

Reply:
You are right, it has to be xx.xx I thought I had used other values before...

Also on Switch, it is limited to fifteen digits.

Fixed in cffe01217f861df07a